### DISCUSSION TITLE
[Help] Bilibili's Policy Infringement on OBS

### DISCUSSION THREAD

**Discussion by Giresharu** [obsproject/obs-studio#9650] [General]
There is a live streaming website, live.bilibili.com, that has developed a piece of software (download link: https://dl.hdslb.com/bili/bililive/win/Livehime-Win-beta-4.52.0.5253-x64.exe) which is profiting from the unauthorized use of your code, without adhering to the GPL 2.0 license.

Furthermore, they have issued a new policy that mandates the use of this software for streaming on their platform, causing outrage among many streamers. This is undoubtedly a very despicable form of monopoly. However, our resources are limited, and perhaps your legal action can assist us and prevent Bilibili from infringing on OBS.

By the way, Bilibili is a very large video website with a significant user base, and it has also been listed in the United States. I'm not very knowledgeable about legal matters, so I'm not sure if this information is helpful to you.

**Comment by copyliu** [comment]
you means https://github.com/bilibili/biliobs ?

**Reply by Giresharu** [reply]
Yes, but not entirely accurate. This repository hasn't been updated for 7 years, while their closed-source version, which they release on their website, has been continuously updated.

**Comment by Fenrirthviti** [comment]
We are aware of this fork already, but thank you for the additional context. We'll look in to this further.
